Source/Lib/Common/ASM_AVX2/EbIntraPrediction_Intrinsic_AVX2.c

```diff
@@ -1151,7 +1154,7 @@ void intra_mode_vertical_luma_avx2_intrin(
     uint8_t            *ref_samples,             //input parameter, pointer to the reference samples
     uint8_t            *prediction_ptr,          //output parameter, pointer to the prediction
     const uint32_t      prediction_buffer_stride, //input parameter, denotes the stride for the prediction ptr
-    const EbBool     skip                    //skip one row
+    const EbBool     skip                    //skip one row 
```

Comment:
```suggestion
    const EbBool     skip                    //skip one row
```
please run
`sed -i 's/[[:space:]]*$//' Source/Lib/Common/ASM_AVX2/EbIntraPrediction_Intrinsic_AVX2.c`